Source/WebCore/dom/Observable.h

```diff
@@ -62,6 +62,10 @@ class Observable final : public ScriptWrappable, public RefCounted<Observable> {
 
     Ref<Observable> drop(ScriptExecutionContext&, uint64_t);
 
+    // Promise-returning operators.
```

Comment:
Let's see, there may be conflicts with https://github.com/WebKit/WebKit/pull/32099. But should be okay, Ill resolve resolve with whatever lands first.